pandas/tests/tools/test_to_datetime.py

```diff
@@ -858,22 +858,31 @@ def test_to_datetime_dt64s_and_str(self, arg, format):
         "dt", [np.datetime64("1000-01-01"), np.datetime64("5000-01-02")]
     )
     def test_to_datetime_dt64s_out_of_bounds(self, cache, dt):
-        msg = "^Out of bounds nanosecond timestamp: .*, at position 0$"
-        with pytest.raises(OutOfBoundsDatetime, match=msg):
-            to_datetime(dt, errors="raise")
+        # We cast to the nearest supported reso, i.e. "s"
+        ts = to_datetime(dt, errors="raise", cache=cache)
+        assert isinstance(ts, Timestamp)
+        assert ts.unit == "s"
+        assert ts.asm8 == dt
+
+        ts = to_datetime(dt, errors="coerce", cache=cache)
+        assert isinstance(ts, Timestamp)
+        assert ts.unit == "s"
+        assert ts.asm8 == dt
```

Comment:
why is checking with both `raise` and `coerce` necessary, given that no errors are thrown in this test anyway? `errors='ignore'` works too

could we either parametrise over that too, or just remove `errors=`?

Reply:
gentle reminder